packages/jellyfish-block/__tests__/rpc/block.test.ts

```diff
@@ -0,0 +1,97 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { Testing } from '@defichain/jellyfish-testing'
+import { CBlock } from '@defichain/jellyfish-block'
+import { SmartBuffer } from 'smart-buffer'
+import BigNumber from 'bignumber.js'
+import { OP_CODES } from '@defichain/jellyfish-transaction'
+
+describe('Block', () => {
+  const testing = Testing.create(new MasterNodeRegTestContainer())
+
+  beforeAll(async () => {
+    await testing.container.start()
+  })
+
+  afterAll(async () => {
+    await testing.container.stop()
+  })
+
+  /**
+   * Wait for block hash to reach a certain height
+   */
+  async function waitForBlockHash (height: number): Promise<string> {
+    await testing.container.waitForBlockHeight(height)
+    return await testing.rpc.blockchain.getBlockHash(height)
+  }
+
+  it('should getBlock with verbosity=1 and have same data as CBlock composer to object', async () => {
+    const blockHash = await waitForBlockHash(1)
+    const hexData = await testing.rpc.blockchain.getBlock(blockHash, 0)
+    const buffer = SmartBuffer.fromBuffer(Buffer.from(hexData, 'hex'))
+    const composable = new CBlock(buffer)
+
+    const block = await testing.rpc.blockchain.getBlock(blockHash, 1)
+
+    const composableObject = composable.toObject()
+    expect(composableObject.blockHeader.version).toStrictEqual(block.version)
+    expect(composableObject.blockHeader.hashPrevBlock).toStrictEqual(block.previousblockhash)
+    expect(composableObject.blockHeader.hashMerkleRoot).toStrictEqual(block.merkleroot)
+    expect(composableObject.blockHeader.hashMerkleRoot).toStrictEqual(block.tx[0])
+    expect(composableObject.blockHeader.time).toStrictEqual(block.time)
+    expect(composableObject.blockHeader.bits).toStrictEqual(parseInt(block.bits, 16))
+    expect(composableObject.blockHeader.height).toStrictEqual(new BigNumber(block.height))
```

Comment:
```suggestion
    expect(composableObject.blockHeader.height).toStrictEqual(new BigNumber(block.height))
    expect(composableObject.blockHeader.stakeModifier).toStrictEqual(block.stakeModifier)
    expect(composableObject.blockHeader.mintedBlocks).toStrictEqual(new BigNumber(block.mintedBlocks))
```

Reply:
The interface from the composer and the one from the RPC don't match exactly. 
The RPC does not send the mintedBlocks nor the stakeModifier.

Reply:
Actually `getblock` RPC returns those fields too. refer https://github.com/DeFiCh/ain/blob/6059de0c533710e7f0214128719201a91230bba1/src/rpc/blockchain.cpp#L148
What it does not return is the `signature` field. not sure why, but better to check with C++ side @Jouzo